lib/formtastic/inputs/base/validations.rb

```diff
@@ -208,6 +204,19 @@ def readonly_attribute?
         def readonly_from_options?
           options[:input_html] && options[:input_html][:readonly]
         end
+
+        private
+
+        def option_value(option, object)
```

Comment:
Would be good to have a comment to point to Rails source code where this is implemented.